src/BaseServiceProvider.php

```diff
@@ -111,6 +113,11 @@ public function registerAdminMiddleware(Router $router)
         }
     }
 
+    public function registerCustomAuthGuard()
+    {
+        View::composer('backpack::*', \Backpack\Base\app\Http\ViewComposers\AuthComposer::class);
```

Comment:
Hmm... I think a more Laravely way to go would be a Facade. Something like ```AdminAuth``` or ```BackpackAuth```?